aiida/engine/daemon/execmanager.py

```diff
@@ -41,6 +41,9 @@ def upload_calculation(node, transport, calc_info, folder, inputs=None, dry_run=
     from tempfile import NamedTemporaryFile
     from aiida.orm import load_node, Code, RemoteData
 
+    debug_printing = False
+    if debug_printing: print('Je commence')
```

Comment:
I realize that this is just for your debugging purposes, but you might want to just the built in logging for this. Simply turn these in `logger.debug('msg')` statements and set `verdi config logging.aiida_loglevel DEBUG` and presto

Reply:
So, what would be the advantage of this? I understand that in the production version having a log for the user is better than just messages popping out everywhere and having the code full of dirty if prints, but if these messages are going to be taken out is there any advantage of using the logger?